lib/resources/example_json/r4_examples_procedure.rb

```diff
@@ -0,0 +1,215 @@
+module Cerner
+  module Resources
+
+    R4_PROCEDURE_BUNDLE ||= {
```

Comment:
The spacing in this file looks off. Did you run it through Beautify?

Reply:
I don't why but only for this file my editor counted 1 tab = 4 spaces. Updated [here](https://github.com/cerner/fhir.cerner.com/pull/322/commits/783dce5f317c0ee97b87117e79fb62dcbca23583#diff-9cc8375943437b05da52e102f3785695R9).

Reply:
+1